utils/utils/src/zod-validators.ts

```diff
@@ -83,6 +83,22 @@ export const $SubmittedLetterTemplate = z.intersection(
   $LetterTemplate
 );
 
+export const $LargePrintLetterTemplate = z.intersection(
+  $LetterTemplate,
+  z.object({
+    letterType: z.literal('x1'),
+  })
+);
+
+export const $ForeignLanguageLetterTemplate = z.intersection(
+  $LetterTemplate,
+  z.object({
+    language: z.string().refine((lang) => lang !== 'en', {
```

Comment:
In `lambdas/backend-client/src/schemas/template.ts` there's a `$Language` validator.  Could use that here (if not creating a circular dependency) and do `language: $Language.exclude(['en'])` here maybe.

Reply:
And probably similar above - do `$LetterType.extract(['x1'])`